opentech/apply/funds/models.py

```diff
@@ -143,12 +158,60 @@ def process_form_submission(self, form):
                 response = cleaned_data.pop(field.id)
                 cleaned_data[field.block.name] = response
 
+        if cleaned_data.get('email'):
+            # TODO: link with active submission user.
+            User = get_user_model()
+            email = cleaned_data.get('email')
+            full_name = cleaned_data.get('full_name')
+
+            try:
+                user_data = {
+                    'username': full_name if full_name else email,
+                    'email': email,
+                    'is_active': False,  # User needs to activate account
+                }
+
+                if full_name and hasattr(User, 'full_name'):
+                    user_data['full_name'] = full_name
+
+                user = User.objects.create_user(**user_data)
+
+                if user:
+                    # TODO: move to method on User?
+                    from opentech.apply.users.utils import send_activation_email
+                    send_activation_email(user)
+
+            except IntegrityError:
+                pass
+
+            self.send_confirmation_email(form, cleaned_data)
+
         return self.get_submission_class().objects.create(
             form_data=cleaned_data,
             page=self.get_parent(),
             round=self,
         )
 
+    def send_confirmation_email(self, form, cleaned_data):
```

Comment:
might be worth making this generic as we will need to apply this to labs. Can do this at a later date.